versions/3.0.md

```diff
@@ -193,7 +193,7 @@ This object can be extended with [Specification Extensions](#specificationExtens
 
 #### <a name="oasVersionString"></a>OpenAPI Version String
 
-The version string signifies the version of the OpenAPI Specification that the document complies to.  The format for this string MUST be `major`.`minor`.`patch`.  The `patch` MAY be suffixed by a hyphen and extra alphanumeric characters.
+The version string signifies which OpenAPI Specification version the document complies to.  The format for this string MUST be `major`.`minor`.`patch`.  The `patch` MAY be suffixed by a hyphen and extra alphanumeric characters.
```

Comment:
Should we say:

The format for this string MUST be a [Semantic Versioning 2.0.0](http://semver.org/) version number in the form `major`.`minor`.`patch` or pre-release version number in the form in the form `major`.`minor`.`patch`.`tags`.

(note: in semver, the optional suffix after the `patch` can contain periods, not just alphanumeric characters.)